code/kg2/build-kg2.sh

```diff
@@ -1,326 +1,336 @@
-#!/usr/bin/env bash
```

Comment:
Not sure why github is showing this entire file as deleted and re-added; changes were made with Dr. Ramsey, and I have also highlighted them with comments in this PR